src/components/__tests__/header.test.tsx

```diff
@@ -4,6 +4,7 @@ import { shallow } from 'enzyme';
 
```

Comment:
It would be nice to up the function coverage on header into green.

Reply:
Coverage for the header is now 100%

Reply:
Prima.